src/common/FundFilter/FundFilterContainer.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { stripesConnect } from '@folio/stripes/core';
+import PropTypes from 'prop-types';
+
+import FundFilter from './FundFilter';
+import { getFundOptions } from '../utils';
+
+const FundFilterContainer = ({ funds, ...rest }) => {
+  const options = getFundOptions(funds);
+
+  return (
+    <FundFilter
+      {...rest}
+      funds={options}
+    />
+  );
+};
+
+FundFilterContainer.propTypes = {
+  funds: PropTypes.arrayOf(PropTypes.object),
```

Comment:
actually. that was the point - to not use passing prop and load it in the container here. Please look at the `resources` prop to pass funds from back-end to the presentation component

Reply:
Done.
- added `manifest`

- used `resources` prop to pass funds from back-end

- updated `getFundOptions` method regarding components updates